### PR TITLE
[nearlib] new account creation tweak

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -251,10 +254,11 @@ export class App extends React.Component<AppProps, AppState> {
       this.toggleWorkspaceSplit();
     }
     if (!this.state.accountId) {
-      const createAccountResponse = await createAccount();
-      const key = createAccountResponse["key"];
-      this.state.keyStore.setKey(createAccountResponse["account_id"], key);
-      App.setAccountId(createAccountResponse["account_id"]);
+      const randomSuffix = Math.floor(Math.random() * 9999999999);
```
Comment (vgrichina) [inline src/components/App.tsx]: 1. should we better just use public key suffix?
2. I feel like it makes sense to have "create account with random key and return account id" logic packaged into Nearlib, as it looks like would be common when testing.

File: src/components/App.tsx [hunk 2]
```diff
@@ -93,6 +93,9 @@ import { StatusBar } from "./StatusBar";
 import { publishArc, notifyArcAboutFork } from "../actions/ArcActions";
 import { RunTaskExternals } from "../utils/taskRunner";
 import * as BrowserLocalStorageKeystore from "nearlib/signing/browser_local_storage_keystore";
+import * as KeyPair from "nearlib/signing/key_pair";
+
+const UUID = require("uuid/v4");
```
Comment (vgrichina) [inline src/components/App.tsx]: looks like UUID is not used

File: src/components/App.tsx (hunk 1 above)
Comment (janedegtiareva) [reply src/components/App.tsx]: Public key would make the name too long. (We have a length limit currently)

File: src/components/App.tsx (hunk 2 above)
Comment (janedegtiareva) [reply src/components/App.tsx]: removed

**Pull request merged**